docs/implementing_RAP/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: How to RAP
+
+tags: 
+  - Implementing RAP
+---
+
+#
+
+!!! tip "TLDR"
+    - Start by doing a skills assessment and identifying training needs
+    - Then map out the process you want to create or replace
+    - Decide on a "thin slice" - a single figure from a single output that you aim to produce first
+    - Create an effective workflow with good coding practices, tests, and version control
+    - Get to work and start coding up the thin slice
+    - Expand the thin slice to complete the pipeline
+
+??? question "Why should we care?"
+    - A little planning can go a long way!
+    - Unless your team already has a high level of coding skill, the RAP process is going to be largely about upskilling and learning good working practices
+    - This process is conducive to creating good working practices, identifying problem points early, and building solid documentation
+
+??? success "Pre-requisites"
+
+    |Pre-requisite | Importance | Note |
+    |--------------|------------|------|
+    | [What is RAP](../introduction_to_RAP/what_is_RAP.md) | Essential | Get to grips with the RAP principles and and why they are important before you dive in |
+
+So you want to implement the RAP principles in your own team?
+
+That's great! This section provides guidance to teams who are looking to start working in the RAP way.
+
+This advice is based on the experiences of the NHS England RAP Squad. They have been working with teams to "RAPify" their processes since around 2021. 
+
+Here's the recommended approach:
+
+## 1) Skills Assessment
```

Comment:
Visually I don't really like the numbers. I'm not sure why, might be the ) A dot after might work better. I think the headings could also work as the links instead of having them below, since each section is pretty short.